cratedb_toolkit/testing/testcontainers/cratedb.py

```diff
@@ -51,8 +56,13 @@ class CrateDBContainer(KeepaliveContainer, DbContainer):
     CRATEDB_PASSWORD = os.environ.get("CRATEDB_PASSWORD", "")
     CRATEDB_DB = os.environ.get("CRATEDB_DB", "doc")
     KEEPALIVE = asbool(os.environ.get("CRATEDB_KEEPALIVE", os.environ.get("TC_KEEPALIVE", False)))
+    CMD_OPTS = {
+        "discovery.type": "single-node",
+        "cluster.routing.allocation.disk.threshold_enabled": False,
```

Comment:
This line accidentally slipped in from my personal workstation configuration, because, well, disk usage is always >80% here ;]. May I humbly ask you to remove it from this patch?